sdk/core/azure-core/src/base64.cpp

```diff
@@ -64,7 +64,7 @@ namespace Azure { namespace Core {
     bio = BIO_push(BIO_new(BIO_f_base64()), bio);
     BIO_set_flags(bio, BIO_FLAGS_BASE64_NO_NL);
     BIO_write(bio, data.data(), static_cast<int>(data.size()));
-    BIO_flush(bio);
+    (void)BIO_flush(bio);
```

Comment:
This return shouldn't be removed.  We need to check the return codes and act accordingly.  I've opened a new issue to remove this dependency all out.
https://github.com/Azure/azure-sdk-for-cpp/issues/2790

Reply:
yeah, I kind of remember this was mentioned before. I think @ahsonkhan moved this from storage to core and would be eventually be replaced for the base64 implementation that @antkmsft  wrote for C99.  If that's still the plan, the change could be targeted for Sep release maybe.